scripts/Test.ps1

```diff
@@ -55,7 +64,24 @@ if(!$testProjects) {
     throw "Failed to find any test projects under: $Path"
 }
 
+if($VSTestHostDebug) {
+    # https://github.com/Microsoft/vstest-docs/blob/master/docs/diagnose.md#debug-test-platform-components
```

Comment:
I typically like a short explanation before providing links in comments.
e.g. Setting this environment variable is necessary for debugging unit tests in Visual Studio Code. See the following for more information:

Reply:
I will add a comment